docs/data-sources/pubsub_subscription.md

```diff
@@ -39,21 +39,50 @@ data "aws_iam_policy_document" "mypolicy" {
 
 ### Optional
 
-- `aws_sqs` (Attributes) Encore provisioned SQS Queue information (see [below for nested schema](#nestedatt--aws_sqs))
 - `env` (String) The environment of the Encore resource. Defaults to the provider environment
-- `gcp_pubsub` (Attributes) Encore provisioned PubSub Subscription information (see [below for nested schema](#nestedatt--gcp_pubsub))
+
+### Read-Only
+
+- `aws_sqs` (Attributes) (see [below for nested schema](#nestedatt--aws_sqs))
+- `gcp_pubsub` (Attributes) (see [below for nested schema](#nestedatt--gcp_pubsub))
 
 <a id="nestedatt--aws_sqs"></a>
 ### Nested Schema for `aws_sqs`
 
 Read-Only:
 
 - `arn` (String) The [ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) for this sqs queue
+- `topic` (Attributes) (see [below for nested schema](#nestedatt--aws_sqs--topic))
+
+<a id="nestedatt--aws_sqs--topic"></a>
+### Nested Schema for `aws_sqs.topic`
+
+Read-Only:
+
+- `arn` (String) The [ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) for this  sns topic
```

Comment:
Should be `sqs`, not `sns`

Reply:
+1

Reply:
I changed my mind. This is the sns topic this sqs queue is subscribed to